sklearn/metrics/scorer.py

```diff
@@ -298,6 +312,8 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
     >>> grid = GridSearchCV(LinearSVC(), param_grid={'C': [1, 10]},
     ...                     scoring=ftwo_scorer)
     """
+    if name is None:
+        name = score_func.__name__
```

Comment:
Indeed the default value should have been `'custom'` instead of `None` as discussed before...